lab5.py

```diff
@@ -0,0 +1,103 @@
+from enum import Enum
+
+class City(Enum):
+    LVIV = 1
+    KYIV = 2 
+    BURSHTYN = 3
+
+class WeatherType(Enum):
+    SUNNY = 0
+    CLOUDY = 1
+    RAINY = 2
+    FOGGY = 3
+
+class Weather(object):
+
+    def __init__(self, day, city, country, temp, humidity, wind_speed, weather_type):
+        self.day = day
+        self.city = city
+        self.country = country
+        self.temp = temp
+        self.humidity = humidity
+        self.wind_speed = wind_speed
+        self.weather_type = weather_type
+        
+    def __del__(self):
+        print("Cleanup has been completed")
+        
+    def getAll(self):
+        """
+        Explanatiom huy na blude

```

Comment:
```suggestion
        Explanation of all parameters

```

Reply:
Thanks for noticing it. I will not commit your suggestion as I reckon it will be better to remove this docstring altogether